inginious/frontend/parsable_text.py

```diff
@@ -334,6 +350,69 @@ def hdl_to_html_block(self, test_id, result, test_case, input_sample, debug_info
             html_block = html_block.replace("updateDiffBlock", "updateWaveDromBlock")
         return html_block
 
+    def client_grader_result_to_html_block(self, test):
+        """Parse each test of client grader submissions into html"""
+
+        test_functions = json.loads(test["functions"].replace("'", "\""), strict=False)
+        test_variables = json.loads(test["variables"].replace("'", "\""), strict=False)
+
+        id_num = ""
+        for i in range(0, len(test["id"])):
+            if test["id"][i].isdigit():
+                id_num += " " + test["id"][i:]
+                break
```

Comment:
What if `test["id"]`is only made of alphabetic characters? Is it possible? In that case the result in `id_num` will be `''`.